genie/validate.py

```diff
@@ -12,31 +12,108 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+class Validator(object):
```

Comment:
this should be an `ABCMeta` class.

Reply:
`ValidateHelper`? `ValidatorWrapper`?

Reply:
Interim solution is to make this like an `ABCMeta` class, and make a separate `GenieValidatorHelper` that inherits from it.